src/Helpers/LocalMachineHelper.php

```diff
@@ -239,6 +240,10 @@ public static function getHomeDir(): string {
     return $home;
   }
 
+  public static function getCloudConfigFilepath(): string {
+    return self::getHomeDir() . '/.acquia/cloud_api.conf';
```

Comment:
We may need to define a constant for `'/.acquia/cloud_api.conf'` so that we can easily set a different file path during testing. Maybe you've already solved that, though.

Reply:
I added some parameters to services.yml, including `app.data_dir`, which is I think what you want. We already inject a custom directory for testing.